src/util/sample.cpp

```diff
@@ -11,11 +11,13 @@ typedef qint32 int32_t;
 #endif
 
 // LOOP VECTORIZED below marks the loops that are processed with the 128 bit SSE
-// registers as tested with gcc 4.6 and the -ftree-vectorizer-verbose=2 flag on
+// registers as tested with gcc 7.5 and the -ftree-vectorize -fopt-info-vec-optimized flags on
 // an Intel i5 CPU. When changing, be careful to not disturb the vectorization.
 // https://gcc.gnu.org/projects/tree-ssa/vectorization.html
 // This also utilizes AVX registers when compiled for a recent 64-bit CPU
 // using scons optimize=native.
+// "SINT i" is the prefered loop index type that should allow vectoszation in
```

Comment:
typo

Reply:
2 typos

Reply:
![](https://clipartart.com/images/the-count-from-sesame-street-clipart-png-6.jpg)